manager/controllers/app/fybrik_notebook_test.go

```diff
@@ -150,7 +150,7 @@ func TestS3Notebook(t *testing.T) {
 	fmt.Printf("blueprint namespace notebook test: %s\n", blueprintNamespace)
 
 	// Forward port of arrow flight service to local port
-	svcName := strings.Replace(application.Status.AssetStates["fybrik-notebook-sample/data-csv"].Endpoint.Hostname, "."+blueprintNamespace+".svc.cluster.local", "", 1)
+	svcName := strings.Replace(application.Status.AssetStates["fybrik-notebook-sample/data-csv"].Endpoint.Hostname, "."+blueprintNamespace, "", 1)
```

Comment:
Why do we need to add anything to Endpoint.Hostname? I think it already includes the namespace